pkg/cmd/init_test.go

```diff
@@ -0,0 +1 @@
+package cmd_test
```

Comment:
is this todo?

Reply:
This file is required to properly measure test coverage. However, it does't make much sense to test the init.cmd code with unit tests because we'll end up mocking Cobra library. So this is to generate coverage report for integration tests runner, which eventually is merged with unit tests runner's report.